routes/nickroutes.js

```diff
@@ -5,50 +5,95 @@ import chalk from 'chalk';
 
 import db from '../database/initializeDB.js';
+import awardMapCustom from '../controllers/awardsController.js';
 
 const router = express.Router();
 
+function getIDByValue(object, value) {
+  return object.filter((item) => item.genre === value);
+}
+
+function getTableRows(table) {
+  return `SELECT * FROM ${table}`;
+}
+
+
+
 /// /////////////////////////////////
 /// ////awards Endpoints////////
 /// /////////////////////////////////
 router.route('/awards')
-  .get(async (req, res) => {
+  .get(async(req, res) => {
     try {
+      const result = await db.sequelizeDB.query(awardMapCustom, {
+        type: sequelize.QueryTypes.SELECT
+      });
       console.log('touched /awards with GET');
-      res.json({message: 'touched /awards with GET'});
-    } catch (error) {
-      console.log(error);
-      res.json({error: 'Something went wrong on the server'});
+      res.json(result);
+    }
+    catch (error) {
+      console.error(error);
+      res.send('Something went wrong on /awards at GET');
     }
   })
-
-  .put(async (req, res) => {
+  .put(async(req, res) => {
     try {
-      console.log('touched /awards with PUT');
-      res.json({message: 'touched /awards with PUT'});
+      const awards = await db.sequelizeDB.query(getTableRows('awards'), {
+        type: sequelize.QueryTypes.SELECT
+      });
+      const awardStatement = `SELECT * FROM awards WHERE award_title = "${req.body.award_title}"`;
+      const selectedMovie = await db.sequelizeDB.query(awardStatement, {
+        type: sequelize.QueryTypes.SELECT
+      });
+      const awardId = awardName.map((movAward) => movAenre.award_id)[0];
+      const updateStatement = `UPDATE awards
+        SET award_title = '${req.body.award_title}', award_id = ${awardId}
+        WHERE award_id = '${awardId}' `;
+      await db.sequelizeDB.query(updateStatement, {
+        type: sequelize.QueryTypes.UPDATE
+      });
+      res.send(`"${req.body.award_title}" Successfully Updated`);
     } catch (error) {
       console.log(error);
-      res.json({error: 'Something went wrong on the server'});
+      res.json({error: 'Something went wrong on the server /awards POST'});
     }
   })
-
-  .post(async (req, res) => {
+  .post(async(req, res) => {
     try {
-      console.log('touched /awards with POST');
-      res.json({message: 'touched /awards with POST'});
-    } catch (error) {
-      console.log(error);
-      res.json({error: 'Something went wrong on the server'});
+      console.log(res.json(req.body));
+      const award = await db.sequelizeDB.query(awardMapCustom, {
+        type: sequelize.QueryTypes.SELECT
+      });
+    
+      const currentID = (await award.length) + 1;
+      const awardTitle = getIDByValue(award_title, req.body.award_title);
+      const awardID = awardTitle.map((awardName) => awardName.award_id)[0];
+      const createStatement = `INSERT INTO awards (award_id, award_title) VALUES (${currentID}, '${req.body.award_title}', ${awardId})`;
```

Comment:
Statement has an extra value being inserted that isn't needed. It should be `INSERT INTO awards (award_id, award_title) VALUES (${currentID}, '${req.body.award_title}')`